test/functional/feature_taproot.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# Test taproot softfork.
```

Comment:
Unify Taproot capitalization in comments.
```suggestion
# Test Taproot softfork.
```

Reply:
Updated.